server/data/cartoons/about-us.json

```diff
@@ -12,11 +12,11 @@
   "buttons": {
     "team": {
       "text": "Meet the Team",
-      "link": "https://mathsoc.uwaterloo.ca/mathsoc-cartoons-team/"
+      "link": "/mathsoc-cartoons-team/"
     },
     "archive": {
-      "text": "Comics Archive",
-      "link": "https://mathsoc.uwaterloo.ca/comics-archive/"
+      "text": "Cartoons Archive",
+      "link": "/comics-archive/"
```

Comment:
this link too, is this supposed to be /cartoons/archive ? (from pages.json)

Reply:
oops thanks for the close attention to detail